docs/suspensive.org/src/components/Sandpack/baseTemplate.ts

```diff
@@ -24,7 +24,7 @@ export default function App() {
     },
   },
   dependencies: {
-    '@tanstack/react-query': '^4.0.0',
+    '@tanstack/react-query': '^4',
```

Comment:
@manudeli https://github.com/toss/suspensive/pull/995#discussion_r1657561711

I'm curious if all the features of Suspensive are intended for TanStack Query version 4. 🤔
Are there any features in Suspensive that only work with TanStack Query version 5?

Reply:
> I'm curious if all the features of Suspensive are intended for TanStack Query version 4. 🤔

Yes! but we support @tanstack/react-query@5 too at once by @suspensive/react-query's postinstall script made by @gwansikk. Check these Pull Request out to know how automatic targeting TanStack Query work please if you need.

- https://github.com/toss/suspensive/pull/953
- https://github.com/toss/suspensive/pull/976

When we install, @suspensive/react-query's postinstall script change target of @suspensive/react-query's dist/index.cjs like below

#### if @tanstack/react-query@4' is installed, @suspensive/react-query target @suspensive/react-query-4
```js
dist/index.js
import * from '@suspensive/react-query-4' // default
```

#### if @tanstack/react-query@5' is installed, @suspensive/react-query target @suspensive/react-query-5
```js
dist/index.js
import * from '@suspensive/react-query-5' // changed by postinstall script
```

> Are there any features in Suspensive that only work with TanStack Query version 5?

We support interface like below with following TanStack Query's version

#### @suspensive/react-query with TanStack Query version 4
- useSuspenseQuery
- useSuspenseQueries
- useSuspenseInfiniteQuery
- queryOptions
- SuspenseQuery
- SuspenseQueries
- SuspenseInfiniteQuery
- QueryErrorBoundary

#### @suspensive/react-query with TanStack Query version 5
- SuspenseQuery
- SuspenseQueries
- SuspenseInfiniteQuery
- QueryErrorBoundary


### Conclusion

In my opinion, We should provide Sandpack with TanStack Query v4 yet. but if you disagree it, suggestion also absolutely good for me! Suggest better solution if you have please.

Reply:
And if you want to get help to know how to use cli of Suspensive React Query.
After you install @suspensive/react-query, you can use `npx suspensive-react-query -h` or `npx srq-h` made by @gwansikk in [this Pull Request with commander.js](https://github.com/toss/suspensive/pull/976)

Reply:
> In my opinion, We should provide Sandpack with TanStack Query v4 yet. but if you disagree it, suggestion also absolutely good for me! Suggest better solution if you have please.

The features we offer that are available in @tanstack/react-query@5 can also be used in @tanstack/react-query@4! In that case, it seems we can keep the Sandpack dependency fixed at @tanstack/react-query@4 👍

Reply:
> In that case, it seems we can keep the Sandpack dependency fixed at @tanstack/react-query@4 👍

That's right~!